lua/entities/gmod_wire_light.lua

```diff
@@ -170,7 +173,7 @@ function ENT:Directional( On )
 end
 
 function ENT:Radiant( On )
-	if On then
+	if On and self:GetOn() then
```

Comment:
There shouldn't be a GetOn check here, this is a function for creating the entity.